portfolio/src/main/webapp/script.js

```diff
@@ -18,9 +18,23 @@
 
 //Get Randome greeting using arrow function -- fetches greeting from server and displays on the page
 function getRandomGreeting() {
-  fetch('/data').then(response => response.text()).then((greeting) => {
-    document.getElementById('greeting-container').innerText = greeting;
+  fetch('/data').then(response => response.json()).then((greeting) => {
+    const listElement = document.getElementById('greeting-container');
+    listElement.innerHTML = '';
+    listElement.appendChild (
```

Comment:
Are there three elements in the list? Perhaps a for-loop from 0 to length(greetings)? That would reduce the need for repeated lines of code, and allow you to expand the list of greetings without having to update this method.

Reply:
Great idea! Making that change!